internal/providers/gofpdf/text.go

```diff
@@ -187,6 +188,33 @@ func (s *text) addLine(textProp *props.Text, xColOffset, colWidth, yColOffset, t
 		return
 	}
 
+	if textProp.Align == align.Justify {
+		const spaceString = " "
+		const emptyString = ""
+
+		text = strings.TrimRight(text, spaceString)
+		textNotSpaces := strings.ReplaceAll(text, spaceString, emptyString)
+		textWidth = s.pdf.GetStringWidth(textNotSpaces)
+		defaultSpaceWidth := s.pdf.GetStringWidth(spaceString)
+		words := strings.Fields(text)
+
+		numSpaces := max(len(words)-1, 1)
+		spaceWidth := (colWidth - textWidth) / float64(numSpaces)
+		x := xColOffset + left
+
+		if isIncorrectSpaceWidth(textWidth, spaceWidth, defaultSpaceWidth, textNotSpaces) {
+			spaceWidth = defaultSpaceWidth
+		}
+
+		for _, word := range words {
+			s.pdf.Text(x, yColOffset+top, word)
+			x += s.pdf.GetStringWidth(word) + spaceWidth
+		}
```

Comment:
If you see the other alignments there are a logic as:
```golang
if textProp.Hyperlink != nil {
    s.pdf.LinkString(dx+xColOffset+left, yColOffset+top-fontHeight, textWidth, fontHeight, *textProp.Hyperlink)
    s.pdf.LinkString(dx+xColOffset+left, yColOffset+top-fontHeight, textWidth, fontHeight, *textProp.Hyperlink)
}
```
This is important to apply a link to text if required, I think that we have to add this here too. Otherwise, we will introduce a bug that links doesn't work with justify.